java/feathub-udf/src/main/java/com/alibaba/feathub/udf/AbstractTimeWindowedAggFunc.java

```diff
@@ -16,10 +16,13 @@
 
 package com.alibaba.feathub.udf;
 
+import org.apache.flink.api.common.typeutils.TypeSerializer;
+import org.apache.flink.api.common.typeutils.base.ListSerializer;
 import org.apache.flink.table.api.DataTypes;
 import org.apache.flink.table.api.dataview.MapView;
 import org.apache.flink.table.catalog.DataTypeFactory;
 import org.apache.flink.table.functions.AggregateFunction;
+import org.apache.flink.table.runtime.typeutils.InternalSerializers;
```

Comment:
This class is @Internal. Is it possible to avoid using internal classes from Flink?

If we have to use an internal class in Flink, it probably suggests Flink's public API lacks the necessary functionalities for users to do the work. Can you describe what is missing here so that we can create a TODO for Flink?

Reply:
I updated the implementation to avoid using internal classes.